packages/performance-tests/test/cases/prepare-fork.ts

```diff
@@ -42,6 +42,23 @@ export default async function prepareFork(sourceDb: BriefcaseDb, addReport: (...
       entityProcessingTimer?.elapsedSeconds ?? -1,
     );
   }
+
+  const targetPath = initOutputFile(`${sourceDb.iTwinId}-${sourceDb.name}-target.bim`, outputDir);
+  const targetDb = SnapshotDb.createEmpty(targetPath, {rootSubject: {name: sourceDb.name}});
```

Comment:
we discussed this, but you should not pass an empty target, but a copied source

Reply:
done